src/seer/automation/autofix/autofix.py

```diff
@@ -333,6 +340,20 @@ def _create_prs(self, title: str, description: str, steps: list[Any], request: A
 
         return prs
 
+    def _get_slug_id(self, organization_id: int) -> str:
```

Comment:
```suggestion
    def _get_slug_id(self, organization_id: int) -> str | None:
```
Hmm I think we should handle this more gracefully. Instead of returning the string `"UNKNOWN"` let's have it return None and skip linking to the issue in the PR, we can use the logic that checks for the issue short id

Reply:
Sounds good.

Reply:
Have made the changes.